library/src/main/java/com/airbnb/android/airmapview/MapboxWebViewMapFragment.java

```diff
@@ -22,4 +22,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
     return view;
   }
+
+  @Override public void setMapType(MapType type) {
+    // TODO: Not implemented
```

Comment:
@felipecsl Yep, Mapbox absolutely supports this feature.  It gets hidden a bit sometimes because MapType is a Google specific name / concept.  Anyway, to swap from Streets to Satellite to Terrain the MapId needs to be changed.  In this case changing the `id` field in [`L.mapbox.map`](https://www.mapbox.com/mapbox.js/api/v2.2.1/l-mapbox-map/) to a corresponding value will change the display.  So the Google to Mapbox conversion looks like:

* NORMAL == `mapbox.streets`
* SATELLITE == `mapbox.satellite`
* TERRAIN == `mapbox.outdoors`